test/Chainweb/Test/Pact/RemotePactTest.hs

```diff
@@ -331,10 +331,14 @@ localPreflightSimTest iot nio = testCaseSteps "local preflight sim test" $ \step
         assertFailure "Preflight /local call produced legacy result"
       Right MetadataValidationFailure{} ->
         assertFailure "Preflight produced an impossible result"
-      Right (LocalResultWithWarns _ ws) -> case ws of
-        [w] | "decimal/integer operator overload" `T.isInfixOf` w ->
-          pure ()
-        ws' -> assertFailure $ "Incorrect warns: " ++ show ws'
+      Right (LocalResultWithWarns cr' ws) -> do
+        -- check the presence of metadata
+        assertBool "Preflight result should have metadata" $ isJust $ _crMetaData cr'
```

Comment:
Should we test exactly what metadata we get here?

Reply:
Existence here matters, otherwise we don't test specific metadata. We _could_ expand it, but i'm satisfied with existence.

Reply:
I'm happier with more coverage, if that's practical. Why not test metadata?

Reply:
We'll do that later, follow-up is on Asana.